src/app/helptext/system/ssh-connections.ts

```diff
@@ -2,37 +2,64 @@ import { T } from '../../translate-marker';
 
 export default {
     name_placeholder: T('Name'),
-    name_tooltip: T(''),
+    name_tooltip: T('Descriptive name of this SSH connection. SSH \
+ connection names must be unique.'),
 
     setup_method_placeholder: T('Setup Method'),
-    setup_method_tooltip: T(''),
+    setup_method_tooltip: T('How to configure the connection\:<br><br> \
+ <i>Manual</i> requires configuring authentication on the remote system. \
+ This can require copying SSH keys and modifying the <i>root</i> user \
```

Comment:
Redundant "require".
```suggestion
 This can include copying SSH keys and modifying the <i>root</i> user \
```